go/libkb/globals.go

```diff
@@ -238,6 +238,7 @@ func (g *GlobalContext) PushShutdownHook(sh ShutdownHook) {
 }
 
 func (g *GlobalContext) ConfigureConfig() error {
+	RemoteSettingsRepairman(g)
```

Comment:
My inclination would be to end the program.  Why keep going if the repairman fails.

Also, will the repairman run every time, even if it's already repaired? 

Also, what stops the repairman from overwriting a good configuration in Local/ with an old configuration in Roaming?